.gitignore

```diff
@@ -21,3 +21,35 @@ output.json
 !.yarn/versions
 
 *.db
+**/*.db
+
+lib-cov
+*.seed
+*.log
+*.csv
+*.dat
+*.out
+*.pid
+*.gz
+*.swp
+
+pids
+logs
+results
+tmp
+
+# API keys and secrets
+.env
+
+# Dependency directory
+node_modules
+
+# OS metadata
+.DS_Store
+Thumbs.db
+
+# Ignore built ts files
+/**/dist/**/*
+
+# ignore yarn.lock
+yarn.lock
```

Comment:
should not ignore lockfile, it's what determines the actual packages versions to pull in